src/style.css

```diff
@@ -1 +1,135 @@
-/* add styles here */
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir';
+  font-weight: 300;
+  font-style: normal;
+  src: url('../src/fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir';
+  font-weight: 600;
+  font-style: normal;
+  src: url('../src/fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+form {
+  display: block;
+  unicode-bidi: isolate;
+}
+
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir', sans-serif;
+  font-size: 16px;
+}
+
+label {
+  cursor: default;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form_big {
+  width: 100%;
```

Comment:
This style should not be doublicated, use .search-form selector to display the style